bindata/network/ovn-kubernetes/common/002-rbac-node.yaml

```diff
@@ -142,6 +142,7 @@ rules:
     - egressips
     - egressqoses
     - egressservices
+    - userdefinednetworks
```

Comment:
This should be under `{{- if .OVN_NETWORK_SEGMENTATION_ENABLE }}`.
We can do that as a follow up

Reply:
oh good point, we won't have this CRD if the feature gate is not enabled.. maybe we should re push?
i don't know what it means to add rbac for a non-existent resource..

Reply:
> i don't know what it means to add rbac for a non-existent resource..
That should be fine, the RBAC doesn't validate that the resource exists already

Reply:
jolly